base/sparse/linalg.jl

```diff
@@ -968,9 +969,8 @@ function Base.cov(X::SparseMatrixCSC, vardim::Int=1; corrected::Bool=true)
     #         [k1*k2*n k2*k2*n]
     # but k1*n = a+b, and k2*n = c+d
     sums = sum(X, vardim)
-    means = sums ./ n
     @inbounds for j in 1:p, i in 1:p
-        part2 = sums[i] * means[j]
+        part2 = sums[i] * sums[j] / n
```

Comment:
Maybe parentheses would be useful around the `/` to prevent overflow on integer matrices? (If multiplication is done first, overflow is much more likely, and computation will use the integer type rather than floating point.)